### ISSUE TITLE
softmax_regression中批量梯度下降问题

### ISSUE THREAD

**Issue by yutian2011** [hzy46/Deep-Learning-21-Examples#16]
写代码时，增加一个epoch的循环，最后的结果只有0.86。为什么大神里面就可以达到0.9多？

    for step in range(500):
        x,y = mnist.train.next_batch(batch_size)
        for i in range(num):#num = mnist.train.num_examples/batch_size
            _,tmp = sess.run([optimizer,loss],{X:x,Y:y}) #optimizer=Adam.minimize(loss)
        #sess.run(optimizer,{X:x,Y:y})

    print("training accuracy",accuracy.eval({X:mnist.train.images,Y:mnist.train.labels}))

**Comment by hzy46** [comment]
你可以调节一下学习率，再看看变化情况

**Comment by yutian2011** [comment]
确实learning_rate的问题。刚开始调试过learning_rate。只是在原有的代码调试了一两个数量级，基本没什么效果。刚才调整了四五个数量级之后，准确率逐步上升。谢谢大神 ；）
